phpstan.neon

```diff
@@ -11,3 +11,5 @@ parameters:
 		- src/
 
 	reportUnmatchedIgnoredErrors: true
+	ignoreErrors:
+		- '#Access to an undefined property Payum\\Core\\Request\\GetHttpRequest::\$headers#'
```

Comment:
Should be added to phpstan baseline. Remove these two lines and run `composer analyse-baseline`